doc/man/scons.xml

```diff
@@ -4569,7 +4610,7 @@ directory as the one the Node represents:
 
 <variablelist>
   <varlistentry>
-  <term><replaceable>d</replaceable>.Dir(<replaceable>name</replaceable>)</term>
+  <term><replaceable>f</replaceable>.Dir(<replaceable>name</replaceable>)</term>
```

Comment:
? if I look at the published manpage, I don't see this having d, but having f? is this fixing a change since the last release?

Reply:
probably... it's not really a functional change anyway, d, f, anything else could be "name of instance".

Reply:
Previously it used f when it was a File node example and d when it was a Dir node.

Reply:
which it's going back to, I agree with you.  I may have done a copy-paste thing when I added markup at some point.